src/test/java/com/bunq/sdk/BunqSdkTestBase.java

```diff
@@ -48,7 +48,7 @@
     protected static final String ATTACHMENT_PATH_IN = "assets/vader.png";
     protected static final String SCHEME_HTTPS = "https://";
     protected static final String URL_PATH_SEPARATOR = "/";
-    protected static final String URL_PATH_SANDBOX_USER = "/sandbox-user";
+    protected static final String URL_PATH_SANDBOX_USER = "/sandbox-user-person";
```

Comment:
@lexym I replaced the sandbox-user endpoint with the sandbox-user-person endpoint.